core/src/main/java/org/apache/iceberg/deletes/Deletes.java

```diff
@@ -62,6 +62,7 @@ public static <T> CloseableIterable<T> filter(
     return equalityFilter.filter(rows);
   }
 
+  /** Returns the same rows that are input, while marking the deleted ones. */
```

Comment:
Maybe this is only me, but when we add a comment,  we can create a full javadoc. I personally hate partial Java doc comments.

Reply:
Fixed.